onnxruntime/test/mlas/bench/bench_qgemm.cpp

```diff
@@ -81,29 +81,37 @@ void QGEMM(benchmark::State& state, bool pack_b) {
 static void QGemmSize(benchmark::internal::Benchmark* b) {
   b->ArgNames(qgemm_arg_names);
   // Args for  "M", "N", "K", "Batch",
-  std::vector<std::vector<int64_t>> mnk_batch = {
-      {512, 64, 512, 12},
-      {512, 512, 64, 12},
-      {512, 768, 768, 1},
-      {512, 768, 3072, 1},
-      {128, 768, 2304, 1},
-      {128, 768, 2304, 6},
-      {128, 1024, 4096, 1},
-      {128, 1024, 4096, 6},
-      {128, 2048, 8192, 1},
-      {128, 4096, 16384, 1}
-  };
-
-  // Args for Threads
-  for (int64_t threads : {2, 4, 8, 16}) {
-    for (auto& shape : mnk_batch) {
-      std::vector<int64_t> copy(shape);
-      copy.push_back(threads);
-      b->Args(copy);
-    }
-  }
+
+  b->Args({512, 32128, 768, 1, 1});
```

Comment:
this will have a conflict with your another pr

Reply:
this has been here for two weeks- come on now